spacelift/data_bitbucket_cloud_integration.go

```diff
@@ -0,0 +1,53 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+var bitbucketCloudFields = struct {
+	USERNAME string
```

Comment:
nit: `Username` for consistency with what you've done in the other data sources?

Reply:
originally all were upper case, golint insisted I don't use UPPER_CASE except in this place 🤔  will change